src/kernel/src/dmem/blockpool.rs

```diff
@@ -0,0 +1,71 @@
+use crate::errno::{Errno, ENOTTY, ENXIO};
+use crate::fs::{IoCmd, VFile, VFileOps};
+use crate::process::VThread;
+use std::num::NonZeroI32;
+use thiserror::Error;
+
+const BLOCKPOOL_FILEOPS: VFileOps = VFileOps {
```

Comment:
Where is this defined in the PS4 kernel?

Reply:
look at the finit call in sys_blockpool open